contrib/lkt/extensions/mains/lkt_toolbox.adb

```diff
@@ -77,6 +92,17 @@ procedure Lkt_Toolbox is
             Print_Diagnostic
               (Diag, Unit,
                Simple_Name (Analysis.Node (S).Unit.Get_Filename));
+
+            --  Raise an error if the declaration including ``Node (S)`` has no
+            --  ``@invalid`` annotation. Update ``Invalid_Decl_Map`` otherwise.
+
+            if Analysis.Node (S).P_Topmost_Invalid_Decl.Is_Null
+            then
+               raise Constraint_Error;
```

Comment:
Should be good now but I couldn't add some tests for those new errors since lkt_toolbox returns a non-zero status code.

Should I add some? This will require to add a new lkt_toolbox option to set the status code, and to modify lkt_driver to manage arguments.

Reply:
I think it’s fine not to add tests, but it’s for sure best to have some: your call!

Reply:
Will keep that for later then!